### DISCUSSION TITLE
dev won't start

### DISCUSSION THREAD

**Discussion by klunkmillan** [wprig/wprig#786] [Q&A]
Hi,
I've been working on an initialized wprig-master; today when I tried to spin it up, it won't start ... I tried npm install then npm run dev, which usually works, and I got an error. So I ran npm run rig-init, then npm run dev, and it seems to start, but then freezes ...  any suggestions? thanks
<img width="1076" alt="Screen Shot 2022-01-27 at 11 05 32 AM" src="https://user-images.githubusercontent.com/89919784/151396929-a85e343d-616c-4c79-a4ba-1ade780f758a.png">
<img width="1004" alt="Screen Shot 2022-01-27 at 11 05 43 AM" src="https://user-images.githubusercontent.com/89919784/151396977-da0fadfb-85e5-41e5-ace0-80a453e57b34.png">
s

**Comment by drikfan** [comment]
I ran into the same issue after updating to wp 5.9 and php8. It seems to be an issue with the phpcs gulp plugin and my config. phpcs seems to be hanging and/or failing silently. I havent had time to look into it really and Im somewhat a noob with phpcs. 
I was able to get up and running again by commenting out the phpcs references in gulp>styles.js and gulp>editorStyles.js. Will need to revisit when im not pressed for time. Hopefully someone who understands this better can point us in the right direction as this is def not a permanent solution.  

in styles.js

![image](https://user-images.githubusercontent.com/165722/151604126-97968670-b931-4f9b-a995-fcd01a094a48.png)

**Reply by klunkmillan** [reply]
ah, I worried it'd be something like that; beyond me; I'll need help :)

**Comment by dthenley** [comment]
Are you running into this issue with a fresh install of WP Rig?
